Makefile

```diff
@@ -266,3 +265,55 @@ release-note:
 .PHONY: generate-release-tags
 generate-release-tags:
 	hack/generate-release-tags.sh
+
+########### Operator SDK Makefile contents
+
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:crdVersions=v1"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=performance-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+# Generate code
+generate-code: controller-gen
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+
+# find or download controller-gen
+# download controller-gen if necessary
+controller-gen:
+ifeq (, $(shell which controller-gen))
+	@{ \
+	set -e ;\
+	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$CONTROLLER_GEN_TMP_DIR ;\
+	go mod init tmp ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
```

Comment:
Can we vendor controller-gen and build it from the vendor directory?
see - https://github.com/openshift/machine-config-operator/blob/master/tools.go
and https://github.com/kubevirt/machine-remediation/blob/master/hack/generate/crds.sh

Reply:
We will need a solution for d/s, thanks, but maybe on a PR on top? It takes an incredible amount of time to rebase it with the ongoing work.- and I am doing it for some time.

Reply:
> @marcel-apf What command do you run that it generates all unneeded manifests, can we limit it and ask to generate only stuff that we currently need?

The first command that creates a new operator-sdk project... : operator-sdk init --domain example.com --repo github.com/example-inc/memcached-operator

But there is no need for that, I can easily remove all unneeded files, I know them by heart :)